.github/workflows/generate-feature-graphic.yml

```diff
@@ -0,0 +1,60 @@
+name: Generate feature graphic
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'fastlane/**'
+jobs:
+  generate-feature-graphic:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install requirements
+      run: |
+        sudo apt update
+        sudo apt install optipng
+        # Install 200 weight versions of relevant Noto (to use for languages not supported by Lexend Deca)
+        sudo apt install fonts-noto-extra fonts-noto-cjk-extra
+        # Custom fonts
+        mkdir "$HOME/.fonts"
+        find .scripts/generate_feature_graphic/fonts -name '*.ttf' -exec cp {} "$HOME/.fonts" \;
+        fc-cache
+    - name: Generate featureGraphic.png for each language
+      run: |
+        for lang in fastlane/metadata/android/*; do
+          pushd "$lang"
+          # Extract text after 'Catima - '
+          subtext=$(grep -oP '(?<=Catima \S ).*' title.txt || true)
+          # Generate temporary .svg with correct text
+          sed "s/Loyalty Card Wallet/$subtext/" ../../../../.scripts/generate_feature_graphic/featureGraphic.svg > featureGraphic.svg
```

Comment:
I assume the text will be unlikely to contain e.g. a `/`, but that would break this.

Reply:
Any suggestion? I feel every character has at least some chance of appearing :(

Reply:
```
export subtext=...
perl -pe 's/foo/$ENV{subtext}/' input > output
```